posthog/session_recordings/test/__snapshots__/test_session_recordings.ambr

```diff
@@ -634,7 +634,7 @@
   FROM "posthog_persondistinctid"
   INNER JOIN "posthog_person" ON ("posthog_persondistinctid"."person_id" = "posthog_person"."id")
   WHERE ("posthog_persondistinctid"."distinct_id" IN ('user2',
-                                                      'user_one_2')
+                                                      'user_one_1')
```

Comment:
Not related I guess? was this intentional?

Reply:
no, it flaps every so often, I bet it'll bounce back on the current run 🙃